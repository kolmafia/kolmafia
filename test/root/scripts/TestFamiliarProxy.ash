familiar f = $familiar[blood-faced volleyball];
            print(f.attributes);
            print(f.block);
            print(f.charges);
            print(f.combat);
            print(f.delevel);
            print(f.drop_item);
            print(f.drop_name);
            print(f.drops_limit);
            print(f.drops_today);
            print(f.elemental_damage);
            print(f.experience);
            print(f.fights_limit);
            print(f.fights_today);
            print(f.hatchling);
            print(f.hp_after_combat);
            print(f.hp_during_combat);
            print(f.image);
            print(f.mp_after_combat);
            print(f.mp_during_combat);
            print(f.name);
            print(f.other_action_after_combat);
            print(f.other_action_during_combat);
            print(f.owner);
            print(f.owner_id);
            print(f.passive);
            print(f.feasted);
            print(f.physical_damage);
            print(f.poke_attribute);
            print(f.poke_level);
            print(f.poke_level_2_hp);
            print(f.poke_level_2_power);
            print(f.poke_level_3_hp);
            print(f.poke_level_3_power);
            print(f.poke_level_4_hp);
            print(f.poke_level_4_power);
            print(f.poke_move_1);
            print(f.poke_move_2);
            print(f.poke_move_3);
            print(f.underwater);
            print(f.variable);
