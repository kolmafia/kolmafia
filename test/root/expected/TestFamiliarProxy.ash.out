eyes
false
0
false
false
none

0
0
false
0
0
0
blood-faced volleyball
false
false
familiar12.gif
false
false

false
false

0
false
false
None
0
3
1
4
1
4
2
Bonk
Frighten
Deluxe Impale
false
false